testing/trino-testing/src/main/java/io/trino/testing/FaultTolerantExecutionConnectorTestHelper.java

```diff
@@ -43,7 +43,7 @@ public static Map<String, String> getExtraProperties()
                 .put("exchange.deduplication-buffer-size", "1kB")
                 .put("fault-tolerant-execution-task-memory", "1GB")
                 // limit number of threads to detect potential thread leaks
-                .put("query.executor-pool-size", "10")
+                .put("query.executor-pool-size", "100")
```

Comment:
not convinced the `100` still guarantees thread leak detection
maybe remove `// limit number of threads to detect potential thread leaks` comment if it is obsolete?

Reply:
You are probably right - but then I think we can just drop this property and keep default 1000